app/src/main/java/nodomain/freeyourgadget/gadgetbridge/model/ActivityUser.java

```diff
@@ -127,11 +136,38 @@ private void fetchPreferences() {
         activityUserYearOfBirth = prefs.getInt(PREF_USER_YEAR_OF_BIRTH, defaultUserYearOfBirth);
         activityUserSleepDuration = prefs.getInt(PREF_USER_SLEEP_DURATION, defaultUserSleepDuration);
         activityUserStepsGoal = prefs.getInt(PREF_USER_STEPS_GOAL, defaultUserStepsGoal);
+        activityUserCaloriesBurnt = prefs.getInt(PREF_USER_CALORIES_BURNT, defaultUserCaloriesBurnt);
+        activityUserDistanceKMeters = prefs.getInt(PREF_USER_DISTANCE_KMETERS, defaultUserDistanceKMeters);
+        activityUserActiveTimeMinutes = prefs.getInt(PREF_USER_ACTIVETIME_MINUTES, defaultUserActiveTimeMinutes);
     }
 
     public Date getUserBirthday() {
         Calendar cal = DateTimeUtils.getCalendarUTC();
         cal.set(GregorianCalendar.YEAR, getYearOfBirth());
         return cal.getTime();
     }
+
+    public int getCaloriesBurnt()
+    {
+        if (activityUserCaloriesBurnt < 0) {
+            activityUserCaloriesBurnt = defaultUserCaloriesBurnt;
+        }
+        return activityUserCaloriesBurnt;
+    }
+
+    public int getDistanceKMeters()
```

Comment:
I already see people asking for fractional values of km... Better use just meters here.

Reply:
Okay, I could change it to meters. But the ZeTime watch goals accepts only kilometers.